File: Nível 2/1044 - Múltiplos/main.py
numeros = input("A: ")

nm1,nm2 = numeros.split()

numero1 = int(nm1)
numero2 = int(nm2)

calculo = numero2 / numero1

if int(calculo) == calculo:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")

File: Nível 2/1044 - Múltiplos/main.lua
numeros = io.read()

x1,x2 = numeros:match("(%S+)%s+(%S+)")
x1,x2 = tonumber(x1), tonumber(x2)

calc = x2 / x1

if math.floor(calc) ~= calc then 
    print("Nao sao Multiplos")
else
    print("Sao Multiplos")
end